lib/ical/stringify.js

```diff
@@ -108,7 +108,15 @@ ICAL.stringify = (function() {
     var jsName = property[0];
     var params = property[1];
 
-    var line = name;
+    var groupName = params['group'];
+    var line;
+    if (groupName) {
+      line = groupName + "." + name;
+    } else {
+      line = name;
+    }
+
+    delete params.group;
```

Comment:
This shouldn't be deleted as the jCal is then changed for future operations. Instead the loop below should do nothing if `paramName` is `group`.